DepositeCalcTests/Pages/SettingsPage.cs

```diff
@@ -0,0 +1,89 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+
+namespace DepositeCalcTests.Pages
+{
+    internal class SettingsPage
+    {
+        private readonly IWebDriver driver;
+        public SettingsPage(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+
+        private IWebElement DateFormatFld => driver.FindElement(By.XPath("//select[@id='dateFormat']"));
+        private IWebElement NumberFormatFld => driver.FindElement(By.XPath("//select[@id='numberFormat']"));
+        private IWebElement DefaultCurrencyFld => driver.FindElement(By.XPath("//select[@id='currency']"));
+        private IWebElement SaveBnt => driver.FindElement(By.XPath("//button[@id='save']"));
+        private IWebElement CancelBtn => driver.FindElement(By.XPath("//button[@id='cancel']"));
+        private IWebElement LogoutLnk => driver.FindElement(By.XPath("//div[@onclick='Logout()']"));
+
+        public LoginPage Logout()
+        {
+            LogoutLnk.Click();
+            return new LoginPage(driver);
+        }
+
+        public CalculatorPage Cancel()
+        {
+            CancelBtn.Click();
+            return new CalculatorPage(driver);
+        }
+
+        public void Save()
+        {
+            SaveBnt.Click();
+            var wait = new WebDriverWait(driver, TimeSpan.FromSeconds(10));
+            wait.Until(drv => IsAlertShown((WebDriver)drv));
```

Comment:
It should be something like this:
```suggestion
            wait.Until(ExpectedConditions.AlertIsPresent);
```

Reply:
This solution returns an error
![image](https://user-images.githubusercontent.com/96728802/200911285-ed6ade81-c7d9-4837-a877-fdf9e03358b4.png)

Reply:
I suggest to use `ExpectedConditions`, as it described in the [answer here](https://stackoverflow.com/questions/60183880/how-to-wait-for-an-alert-in-selenium-webdriver-with-c).